core/include/traccc/ambiguity_resolution/alt_greedy_ambiguity_resolution_algorithm.hpp

```diff
@@ -0,0 +1,50 @@
+/** TRACCC library, part of the ACTS project (R&D line)
+ *
+ * (c) 2024 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+#pragma once
+
+// Project include(s).
+#include "traccc/edm/track_state.hpp"
+#include "traccc/utils/algorithm.hpp"
+#include "traccc/utils/messaging.hpp"
+
+// Greedy ambiguity resolution adapted from ACTS code
+
+namespace traccc {
```

Comment:
Please use the `traccc::host` namespace.

Reply:
Fixed